### PR TITLE
Updated Json.Net versions

### PR DESCRIPTION
**Pull request by ajaybhargavb**
@Eilon

### CODE REVIEW

File: src/Microsoft.AspNetCore.Authentication.Facebook/project.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "Microsoft.AspNetCore.Authentication.OAuth": "1.0.0-*",
-    "Newtonsoft.Json": "6.0.6"
+    "Newtonsoft.Json": "8.0.2"
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication.Facebook/project.json]: You can remove this one, it's transitive from OAuth.

File: src/Microsoft.AspNetCore.Authentication.Cookies/project.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "Microsoft.AspNetCore.Authentication": "1.0.0-*",
     "Microsoft.Extensions.Options": "1.0.0-*",
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
-    "Newtonsoft.Json": "6.0.6"
+    "Newtonsoft.Json": "8.0.2"
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication.Cookies/project.json]: What in cookies is using Newtonsoft?

Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.Authentication.Cookies/project.json]: Nothing. Removed.

File: src/Microsoft.AspNetCore.Authentication.Facebook/project.json (hunk 1 above)
Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.Authentication.Facebook/project.json]: Done.

**Comment by ajaybhargavb** [comment]
:up: :date:

**Comment by Tratcher** [comment]
:shipit:

**Comment by Eilon** [comment]
:shipit: nice, simplifying is good!

**Pull request merged**